modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -84,6 +99,16 @@ function restore_mongo {
     "${tempdir}/${database}"
 }
 
+function dump_elasticsearch {
+  es_dump_restore dump http://localhost:9200/ ${database} ${tempdir}/${filename}
```

Comment:
Hi @rtrinque , 

Sorry, could you please tell us where is **es_dump_restore** defined.

Reply:
`es_dump_restore` is [here](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_elasticsearch/manifests/dump.pp#L11-L15) in Puppet, it's a package.